File: app/src/main/kotlin/com/krsk/qiitareader/presentation/activity/MainActivity.java
package com.krsk.qiitareader.presentation.activity;

import android.app.Fragment;
import android.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.krsk.qiitareader.R;
import com.krsk.qiitareader.presentation.fragment.ItemListFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            final Fragment fragment = ItemListFragment.newInstance();
            final FragmentTransaction fragmentTransaction = getFragmentManager().beginTransaction();
            fragmentTransaction.replace(android.R.id.content, fragment);
            fragmentTransaction.commit();
        }
    }
}

File: app/src/main/kotlin/com/krsk/qiitareader/presentation/activity/MainActivity.kt
package com.krsk.qiitareader.presentation.activity

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.krsk.qiitareader.R
import com.krsk.qiitareader.presentation.fragment.ItemListFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (savedInstanceState == null) {
            val fragment = ItemListFragment.newInstance()
            val fragmentTransaction = fragmentManager.beginTransaction()
            fragmentTransaction.replace(android.R.id.content, fragment)
            fragmentTransaction.commit()
        }
    }
}
